roles/airflow_scheduler/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- include_role:
+    name: .galaxy/ansible-airflow-in-docker
+  vars:
+    airflow_container_name: "{{ aura_airflow_container_name }}"
+    airflow_script_location: "{{ aura_airflow_script_location }}"
+    airflow_data_input_location: "{{ aura_airflow_data_input_location }}"
+    airflow_data_output_success_location: "{{ aura_airflow_data_output_success_location }}"
+    airflow_data_output_failed_location: "{{ aura_airflow_data_output_failed_location }}"
+    airflow_docker_networks:
+      - name: "{{ aura_docker_network }}"
+
+- name: Copy requirements file
+  become: true
+  synchronize:
+    src: files/
+    dest: "{{ aura_airflow_requirements_location }}"
+    delete: yes
+
+- name: Restart container to install the custom requirements
```

Comment:
Passer le role d'écriture côté ansible-docker.

Voir : https://github.com/Aura-healthcare/Aura_infrastructure/blob/master/roles/logstash/tasks/main.yml
et : https://github.com/Aura-healthcare/ansible-logstash-in-docker/blob/master/tasks/main.yml

Reply:
DONE